lib/diagnostic.rb

```diff
@@ -29,7 +29,7 @@
 
 character = 'Jar Jar Binks'
 
-Diagnostic.interpolation = 'ANSWER is really a Sith Lord.'
+Diagnostic.interpolation = '#{name} is really a Sith Lord.'
 
```

Comment:
Here, you just need to interpolate the `character` variable.

`#{character} is really a Sith Lord.`